cpg_infra/billing_aggregator/aggregate/utils.py

```diff
@@ -624,10 +650,28 @@ async def _get_jobs_and_add_to_queue(
         """
         batch_id = batch['id']
         jobs_cnt = batch['n_jobs']
-        batch_name = batch.get('attributes', {}).get('name')
 
-        # check if hail batch query or too many jobs
-        if batch_name is None or jobs_cnt > DEFAULT_MAX_JOBS_PER_BATCH:
+        if batch['cost'] == 0:
+            # add one arbitrary job to keep the logic consistent
+            # with the rest of the code
+            empty_job = {
+                'batch_id': batch_id,
+                'job_id': jobs_cnt,
+                'state': batch.get('state'),
+                'resources': {},
+                'attributes': {},
+                'cost': {
+                    # we need some entry in the cost, so we can insert it
+                    'ip-fee/1024/1': 0,
```

Comment:
Do you want to call this something notable so it's easier to track down in the future, like '<empty-batch-cost-placeholder>' or will that break things?

Reply:
Let me try '', makes more sense.

Reply:
So yes {' ': 0} works.